test/e2e/chaos/basic.go

```diff
@@ -59,581 +59,596 @@ var _ = ginkgo.Describe("[chaos-mesh] Basic", func() {
 		}
 	})
 
-	ginkgo.It("PodFailure", func() {
-		ctx, cancel := context.WithCancel(context.Background())
-		nd := fixture.NewCommonNginxDeployment("nginx", ns, 1)
-		_, err := kubeCli.AppsV1().Deployments(ns).Create(nd)
-		framework.ExpectNoError(err, "create nginx deployment error")
-		err = waitDeploymentReady("nginx", ns, kubeCli)
-		framework.ExpectNoError(err, "wait nginx deployment ready error")
-
-		podFailureChaos := &v1alpha1.PodChaos{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      "nginx-failure",
-				Namespace: ns,
-			},
-			Spec: v1alpha1.PodChaosSpec{
-				Selector: v1alpha1.SelectorSpec{
-					Namespaces: []string{
-						ns,
+	// pod chaos case in [PodChaos] context
+	ginkgo.Context("[PodChaos]", func() {
+
+		// podfailure chaos case in [PodFailure] context
+		ginkgo.Context("[PodFailure]", func() {
+
+			ginkgo.It("[Duration]", func() {
```

Comment:
What's the difference between `Duration` case and `Schedule` case, seems they have almost the same test code, expect the chaos spec is a little different. Can we combine them together into one case?